content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Install Istio with the Istio CNI plugin
-description: Instructions to install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
+description: Install and use Istio with the Istio CNI plugin, allowing operators to deploy services with lower privilege.
```

Comment:
*CNI* and *Container Network Interface* terms are used interchangeably. I'd prefer if we specify *CNI (Container Network Interface)* at the first time and use *CNI* in the rest of the document.

Reply:
@venilnoronha 
I agree but that is outside the scope of the PR and we can fix it post 1.1